spec/say_when/storage/active_record/job_spec.rb

```diff
@@ -48,7 +48,21 @@
     j.next_fire_at.should == ce.next_fire_at
   end
 
+  it "resets acquired jobs" do
+    old = 2.hours.ago
+    j = SayWhen::Storage::ActiveRecord::Job.create!(@valid_attributes.merge({
+      :status => 'acquired', :updated_at => old, :created_at => old
+    }))
+
+    SayWhen::Storage::ActiveRecord::Job.reset_acquired(3600)
+
+    j.reload
+    j.status.should == 'waiting'
+  end
+
   it "can find the next job" do
+    SayWhen::Storage::ActiveRecord::Job.delete_all
```

Comment:
there is not, I guess I was keeping db related stuff in the AR tests, but could put that in.
I'll add a ticket for that, as it is outside the scope of this PR I think.